src/applications/ForgeVTTFilePickerV2.mjs

```diff
@@ -440,9 +440,8 @@ try {
        */
       async _prepareContext(options) {
         const context = await super._prepareContext(options);
-        context.tabs = this.constructor.TABS.sources.tabs;
-        if (ForgeVTT.usingTheForge) {
-          context.tabs = context.tabs.filter((t) => t.id !== "s3");
+        if (ForgeVTT.usingTheForge && context.tabs["s3"]) {
+          delete context.tabs["s3"];
```

Comment:
@wyrmisis Do you happen to recall why we were overwriting `context.tabs`? This seems to have been the source of the issue, without that line the tab lists is properly filtered by Foundry. But I worry there may be some unintended consequences of removing it.